source/_patterns/03-templates/55-monthly-good/01-landing.mustache

```diff
@@ -0,0 +1,9 @@
+<div class="row">
+	<div class="columns monthly-good monthly-good-landing">
+		<p>Monthly Good <strong>landing</strong> page placeholder</p>
+
+		<p class="button"><a href="{{confirm_href}}">Join Monthly Good</a></p>
```

Comment:
When I pulled up this template the font color on the button looked off. Do we need to wrap the <a> in the <p> tag or can we just change it to: 
<a class="button" href="{{confirm_href}}">Join Monthly Good</a>

Reply:
Yikes1 Fixing...

Reply:
Had the button class o the wrong element :0